engine/gameobject/src/gameobject/gameobject_props_lua.cpp

```diff
@@ -223,7 +223,9 @@ namespace dmGameObject
             if (lua_isstring(L, -2))
             {
                 void* userdata = 0;
-                dmhash_t id = dmHashString64(lua_tostring(L, -2));
+                size_t length;
+                const char* string = lua_tolstring(L, -2, &length);
+                dmhash_t id = dmHashBuffer64(string, length);
```

Comment:
Aren't you getting warnings when bulding this, about casting from size_t to uint32_t?
Please avoid adding more warnings to the build.

Reply:
I didn't notice any. But I probably lost them among all the output.